pegasus/cache/i18n/it-IT.yml

```diff
@@ -2332,8 +2340,33 @@ it-IT:
     target='_blank'>Maggiori informazioni</a>
   hoc2018_dance_amazon_learn_more_future_engineer_subheading_title: Ulteriori informazioni
     su Amazon Future Engineer
+  hoc2018_dance_amazon_about_future_engineer_title: Informazioni su Amazon Future
+    Engineer
+  hoc2018_dance_amazon_about_future_engineer_description: Learn about Amazon’s kindergarten
+    to career program to increase access to computer science education. Find information
+    on how to get involved in our summer camps, high school CS classes, scholarships,
+    and internships.
   hoc2018_dance_amazon_about_future_engineer_button: Ulteriori informazioni
+  hoc2018_dance_amazon_try_coding_this_summer_title: Try coding this summer
+  hoc2018_dance_amazon_try_coding_this_summer_description: Get involved in Amazon
+    Future Engineer’s free, fun, and inclusive computer science camps! Summer camps
+    are available for students from kindergarten through 8th grade in multiple locations
+    in the US for summer 2019.
+  hoc2018_dance_amazon_try_coding_this_summer_button: Join a camp
+  hoc2018_dance_amazon_scholarship_programs_title: Scholarship programs
+  hoc2018_dance_amazon_scholarship_programs_description: Amazon is offering scholarships
+    of $10,000 per year for underserved students who have successfully completed or
+    are currently enrolled in an AP CS course and intend to study a CS degree at a
+    4-year college or university in the United States.
   hoc2018_dance_amazon_scholarship_programs_button: Applica ora
+  hoc2018_dance_amazon_internships_title: Come Intern at Amazon
```

Comment:
These are all approved translations that weren't actually translated. It kind of seems like this is on purpose, potentially as these strings are US-centric?

Reply:
That does seem super weird. It was Francesco who did it, though, so I'm going to assume he had a good reason.

Reply:
If I remember correctly, the amazon internship information is only shown when the selected locale is US.